src/main/java/uk/gov/gchq/palisade/exception/PalisadeWrappedErrorRuntimeException.java

```diff
@@ -27,6 +29,7 @@ public PalisadeWrappedErrorRuntimeException(final Error error) {
         this.error = error;
     }
 
+    @Generated
     public Error getError() {
         return error;
     }
```

Comment:
Add toString, hashCode and equals methods

Reply:
See the comment for PalisadeCheckedException.